composer.json

```diff
@@ -20,7 +20,7 @@
         "php": ">=8.0",
         "ext-pdo": "*",
         "ext-curl": "*",
-        "utopia-php/database": "0.38.*"
+        "utopia-php/database": "0.39.*"
```

Comment:
The lock file should be regenerated too.

Reply:
done